src/index.js

```diff
@@ -133,6 +159,7 @@ export default (function create(defaults) {
 		}
 		const options = deepMerge(defaults, config || {});
 		let data = options.data;
+		url = buildUrl(url, options.params, options.paramsSerializer);
```

Comment:
If we go with the smaller variant of buildUrl, it can be inlined here to save a few bytes (versus letting Terser do it):

```suggestion
		if (options.params) {
			const divider = ~url.indexOf('?') ? '&' : '?';
			const query = options.paramsSerializer ?
				options.paramsSerializer(options.params) :
				new URLSearchParams(options.params);
			url += divider + query;
		}
```

---

Another idea (not sure if it'll end up being smaller or larger) might be to have a default value for `options.paramsSerializer` that uses URLSearchParams. On [line 44](https://github.com/developit/redaxios/pull/32/files#diff-1fdf421c05c1140f6d71444ea2b27638R44):

```js
defaults = defaults || {
  paramsSerializer: params => new URLSearchParams(params)
};
```